packages/garbo/src/resources/darts.ts

```diff
@@ -0,0 +1,22 @@
+import { $effect, $item, get, have } from "libram";
+
+export const guaranteedBullseye = () =>
+  get("everfullDartPerks").includes("25% Better bullseye targeting") &&
+  get("everfullDartPerks").includes("25% More Accurate bullseye targeting") &&
+  get("everfullDartPerks").includes("25% better chance to hit bullseyes");
+
+export const safeToAttemptBullseye = () =>
+  guaranteedBullseye()
+    ? have($item`Everfull Dart Holster`)
+    : have($item`Everfull Dart Holster`) && have($item`spring shoes`);
+
+export const canBullseye = () =>
+  guaranteedBullseye()
+    ? have($effect`Everything Looks Red`)
+    : have($effect`Everything Looks Red`) ||
+      have($effect`Everything Looks Green`);
```

Comment:
I would structure these differently: 
```suggestion
export const safeToAttemptBullseye = () =>
  have($item`Everfull Dart Holster`) && (guaranteedBullseye() || have($item`spring shoes`));

export const canBullseye = () =>
  have($effect`Everything Looks Red` ) && (guaranteedBullseye() || !have($effect`Everything Looks Green`));
```